compiler/pkg/openapi_generator/test_data/foos.yaml

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
can this file be multiDoc Yaml

Reply:
@ramramu3433 ,  Multiple CRDs in a file are currently not allowed because the compiler only creates one CRD file per node. However, the compiler generates file with this format (`---`)

Reply:
sure @PavitraS , Thanks for clarification